cmake/projects/ogles_gpgpu/hunter.cmake

```diff
@@ -117,7 +117,18 @@ hunter_add_version(
     "https://github.com/hunter-packages/ogles_gpgpu/archive/v0.2.4.tar.gz"
     SHA1
     ce38c35533fbb8f8efc30eb74d300598ee2132aa
-)
+    )
+
+hunter_add_version(
+    PACKAGE_NAME
+    ogles_gpgpu
+    VERSION
+    0.2.5
```

Comment:
Please add it to `cmake/configs/default.cmake`

Reply:
Oops.  Done.